tests/acceptance/cli_test.py

```diff
@@ -782,6 +782,19 @@ def test_down(self):
         assert 'Removing network v2full_default' in result.stderr
         assert 'Removing network v2full_front' in result.stderr
 
+    def test_down_url(self):
+        self.base_dir = '.'
+        url = 'https://raw.githubusercontent.com/docker/compose/' \
+              '771bf5f7ea059f1efede26bd9b164e05467d289e/tests/fixtures/' \
+              'simple-composefile/docker-compose.yml'
```

Comment:
Going out to a real url may cause this test to be flaky. I think the Engine tests this same functionality by spinning up a local static file server as part of the tests.